docs/conf/portland/2025/cfp.rst

```diff
@@ -64,10 +64,14 @@ We welcome talks from first-time speakers, from industry experts, and from every
 Whatever your background and experience, we prefer hearing about new approaches rather than about tried-and-tested technology.
 We especially welcome talks from underrepresented groups within the tech community.
 We want to hear a variety of viewpoints, so we limit speakers to two talks in any four year period at each location.
-That said, we do strongly prefer talks written and given by a single speaker, and if you're planning to submit with a co-speaker, please let us know why you're doing so.
 
 If you're a WTD conference organizer, please only submit talks to conferences you're not actively organizing.
 
+**People on stage**
+
+We *strongly* prefer talks written and given by one *single speaker*. If you're planning to submit with a co-speaker, please let us know why, and take note that it will affect the chances of your talk being selected.
```

Comment:
```suggestion
We *strongly* prefer talks presented by one *single speaker*. If you're planning to submit with a co-speaker, please let us know why, and take note that it will affect the chances of your talk being selected.
```